### PR TITLE
ci(): adapt build script to rollup

### PR DESCRIPTION
**Pull request by ShaMan123**
change the script a bit to use rollup watch because it is 10 times (4s vs 13-17s) faster than using fs and rebuilding!

### CODE REVIEW

File: scripts/index.js [hunk 1]
```diff
@@ -80,7 +83,37 @@ class ICheckbox extends Checkbox {
 }
 inquirer.registerPrompt('test-selection', ICheckbox);
 
+// async function rollupBuild(options = {}, onComplete) {
```
Comment (ShaMan123) [inline scripts/index.js:86]: we can use the custom watcher in the future if we need

File: scripts/index.js [hunk 2]
```diff
@@ -90,23 +123,41 @@ function build(options = {}) {
...
     if (require(path.resolve(websiteDir, 'package.json')).name !== 'fabricjs.com') {
         console.log(chalk.red('Could not locate fabricjs.com directory'));
     }
-    cp.spawn('npm', ['run', 'start:dev'], {
+    const args = ['run', 'start:dev'];
+
+    //  WSL ubuntu
+    // https://github.com/microsoft/WSL/issues/216 
+    // os.platform() === 'win32' && args.push('--', '--force_polling', '--livereload');
+    if (os.platform() === 'win32') {
```
Comment (ShaMan123) [inline scripts/index.js:154]: poor suckers
me part of them

File: scripts/index.js [hunk 3]
```diff
@@ -364,18 +432,14 @@ program
 program
     .command('build')
     .description('build dist')
-    .option('-f, --fast')
+    .option('-f, --fast', 'skip minifying')
```
Comment (ShaMan123) [inline scripts/index.js:435]: we can negate this to minify
but who cares

**Review by ShaMan123 — COMMENTED** [review]
**QUICK MERGE**

Thanks!

File: scripts/index.js (hunk 2 above)
Comment (asturur) [reply scripts/index.js:154]: Did you try wls2?  Windows linux subsystem...

Comment (asturur) [reply scripts/index.js:154]: Ok is written one line below

**Review by asturur — APPROVED** [review]

File: scripts/index.js (hunk 2 above)
Comment (ShaMan123) [reply scripts/index.js:154]: crossing between kernels is the slowest thing ever 
or something there that slowed down my amazing computer to a halt
I think I should remove it though besides the perf damage it has caused it is really good

**Pull request merged**